Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -432,14 +431,12 @@ public IAsyncLoop Mine(WalletSecret walletSecret)
         /// One a new block is staked, this method is used to verify that it
         /// is a valid block and if so, it will add it to the chain.
         /// </summary>
-        /// <param name="context">Information about the new block.</param>
+        /// <param name="block">The new block.</param>
         /// <param name="chainTip">Block that was considered as a chain tip when the block staking started.</param>
-        private async Task CheckStakeAsync(ContextInformation context, ChainedBlock chainTip)
+        private async Task CheckStakeAsync(Block block, ChainedBlock chainTip)
```

Comment:
got compiler warning `warning CS1998: This async method lacks 'await' operators and will run synchronously.`

Reply:
also i've noticed i've introduced a warning at https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Stratis.Bitcoin.IntegrationTests/CoinViewTests.cs#L66 - `this.` is missing there, could you maybe just add it together with fixing your warning?

Reply:
OK yeah I will fix both cases